Automattic/wp-calypso
# media: track currently active query in fetching reducer

#### Changes proposed in this Pull Request

* Add `setQuery` action to explicitly track currently active media query via `fetching` reducer

#### Testing instructions

* Follow testing instructions from and make sure the following works:

1. Go to the `media` section. Make sure all files are loading.
2. Use the filter bar to change to a certain file type like Images, Documents, etc – make sure that loads as well
3. Being on a certain file type category, hit reload and make sure it loads all the files

Fixes #44149
